components/sbm-recipes-boot-upgrade/src/main/java/org/springframework/sbm/boot/upgrade_27_30/actions/Boot_27_30_JmxEndpointExposureAction.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.sbm.boot.upgrade_27_30.actions;
+
+import org.springframework.sbm.boot.properties.finder.SpringBootDefaultPropertiesFinder;
+import org.springframework.sbm.engine.context.ProjectContext;
+import org.springframework.sbm.engine.recipe.AbstractAction;
+
+import java.util.Optional;
+
+public class Boot_27_30_JmxEndpointExposureAction extends AbstractAction {
+
+    public static final String JMX_ENDPOINT = "management.endpoints.jmx.exposure.include";
+
+    @Override
+    public void apply(ProjectContext context) {
+        SpringBootDefaultPropertiesFinder springBootDefaultPropertiesFinder = new SpringBootDefaultPropertiesFinder();
```

Comment:
Great choice! (I already forgot about this `Finder`) 😬 
You found an elegant way around the limitation that the `SpringBootDefaultPropertiesFinder` works only with single-module applications (for now). Nice! 
I added an issue: #476

Reply:
Thank you so much for it@fabapp2 . I just trying to do go through the flow of code and to understand the code base so  I can contribute much better as I am new to this so that's why it is abit long time for me :) And if possible i can look onto the issue #476 if I find the solution for it .

Reply:
Hi @ishu-thakur 
No problem. Take your time and feel free to ask questions.
Feel free to look into #476. I am not sure how many callers are using the Finder but I think it shouldn't be too many. 
If you want to fix the Finder, could you please provide a test for the finder only in a single (should exist 🤞) and a multi-module project?